e2e/pod.go

```diff
@@ -210,3 +210,24 @@ func deletePodWithLabel(label, ns string, skipNotFound bool) error {
 	}
 	return err
 }
+
+// getMountPath fetches the path on an app where volume is mounted.
+func getMountPath(app *v1.Pod) string {
+	path := app.Spec.Containers[0].VolumeMounts[0].MountPath
+	return path
+}
+
+// calculateMd5Sum returns the md5sum of a file inside a pod.
+func calculateMd5Sum(f *framework.Framework, app *v1.Pod, filePath string, opt *metav1.ListOptions) (string, error) {
+	cmd := fmt.Sprintf("md5sum %s", filePath)
```

Comment:
md5 is not _that_ reliable, consider using `sha512sum` instead

Reply:
Sure, I'll make use of `sha512sum` instead :+1: